Parser/rdiBeam2Earth.m

```diff
@@ -0,0 +1,194 @@
+function [veast,vnrth,wvel,evel]=rdiBeam2Earth(b1,b2,b3,b4,head,pitch,roll,ba,cvxccv,updown, distance)
+% Function to convert ADCP beam velocities to Earth coordinate velocity.
+% b1,b2,b3,b4 are four beam velocities
+% head is the heading reading, in degrees.
```

Comment:
AFAIK the inputs here are not in degrees, since the call to this function is before converting the raw readings to degrees (LSD=0.01 in the manual).

Reply:
This code came from someone else (I think the Scripps group). Possibly their input was in degrees. I hope that I was using degrees to input. Need to check what the parser does (does it do a change when reading into the sample_data structure)?

Reply:
all is fine, it was just a remark because the help is misleading. There is a conversion just below (*0.01), which puzzled me,  until I went to the workhorse manual and found this LSD factor. Which also lead me to think in the possibility of double conversions cases. These are the oddities I need to be careful about since I'm shifting the code calls to do things more in line with what we do in other places.